package.json

```diff
@@ -19,7 +19,7 @@
     "@babel/preset-typescript": "^7.17.12",
     "@react-native-async-storage/async-storage": "^1.17.9",
     "@react-native-community/audio-toolkit": "git+https://github.com/tbenr/react-native-audio-toolkit.git#refs/tags/v2.0.3-status-v6",
-    "@react-native-community/blur": "git+https://github.com/Kureev/react-native-blur#refs/tags/v3.6.1",
+    "@react-native-community/blur": "git+https://github.com/status-im/react-native-blur#refs/tags/v4.3.0-status",
```

Comment:
the react-native-blur library currently doesn't support the `trasnparent` blur type for ios, as mentioned in Issues https://github.com/Kureev/react-native-blur/issues/292 https://github.com/Kureev/react-native-blur/issues/407.
But currently, a [PR](https://github.com/Kureev/react-native-blur/pull/513) is out that adds this functionality. So currently we are forking the library and applying that patch.
This patch works great with several colors, except a few colors like our neutral-100. With our neutral-100 it creates a partially visible blue overlay as shown in the image. Still, it is much better than old blur types like light, and dark. So for now we are using this version with a patch for release and will improve it more later.
_(will create issue for that)_